src/main/java/repositories/ItemRepository.java

```diff
@@ -0,0 +1,52 @@
+package repositories;
+
+import entities.Item;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ItemRepository implements IGenericRepository<Item> {
+
+    Map<Integer, Item> items;
+
+    public ItemRepository() {
+        items = new HashMap<>();
+    }
+
+    @Override
+    public void add(Item item) {
+        items.put(item.getId(), item);
+    }
+
+    @Override
+    public Item update(Item item) {
+        if (item != null && items.containsKey(item.getId())) {
+            return items.replace(item.getId(), item);
+        }
+
+        return null;
+    }
+
+    @Override
+    public Item delete(Integer id) {
+        if (id != null && items.containsKey(id)) {
+            return items.remove(id);
+        }
+
+        return null;
+    }
+
+    @Override
+    public Item find(Integer id) {
+        if (id != null && items.containsKey(id)) {
+            return items.get(id);
```

Comment:
Use getOrDefsult()

Reply:
What should I return by default? an empty Item?